sdk/communication/Azure.Communication.CallAutomation/src/Models/AddParticipantsOptions.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.Communication.CallAutomation
+{
+    /// <summary>
+    /// The add participants operation options.
+    /// </summary>
+    public class AddParticipantsOptions: RepeatabilityHeaders
+    {
+        /// <summary>
+        /// Creates a new AddParticipantsOptions object.
+        /// </summary>
+        /// <param name="participantsToAdd"></param>
+        /// <param name="repeatabilityRequestId"></param>
+        /// <param name="repeatablityFirstSent"></param>
+        public AddParticipantsOptions(IEnumerable<CommunicationIdentifier> participantsToAdd, Guid? repeatabilityRequestId = null, string repeatablityFirstSent = default)
```

Comment:
Optional values to not be included in the constructor, to keep it consistent throughout the sdk.

... required service method parameters are modeled as Options class constructor parameters and get-only properties, and optional parameters are get-set properties.

Reply:
updated. ty!